src/modules/cvs/components/skills/skills.module.scss

```diff
@@ -0,0 +1,22 @@
+.skills {
+    width: 46%;
```

Comment:
Probably could just use something like `min(48rem, 100%)` for the width, so that the content width wouldn't go beyond 48rem (which should be enough for this sort of pages, tested with 'languages' and looks fine)) and would take all the space available on screens smaller than that (-padding for the `main`, which is about 10% of screen width). That would also eliminate the need for some of those (almost empty) media-queries

And for the font-size in various places you could try using smth like `clamp(1rem, 5vw, 1.5rem)`, so that the font would be 1.5rem for large screens, would shrink for smaller sizes but would stop at 1rem. This could also make some of those media-queries redundant